docs/dataformats/observation_lists.rst

```diff
@@ -22,23 +22,48 @@ This has the advantage that everyone can work with whatever tool they like. Here
 
-A run list must have at least a column called ``Run``::
+A run list must have at least a column called ``OBS_ID``::
  
-   Run
+   OBS_ID
    42
    43
 
-Usually it has many more columns with information about each run::
- 
-   Run,Telescope_Pattern,Date,Duration,GLON,GLAT,Target,Zenith
-   1234,24,2013-03-22 14:32,1832,83.7,-5.2,Crab Nebula,32
+Usually it has many more columns with information about each observation. A list of Gammapy supported columns is::
+
+   OBS_ID: observation ID as an integer (starting at 0 or 1? does it matter?)
+   RA: pointing position right ascension in equatorial (ICRS) coordinates
+   DEC: pointing position declination in equatorial (ICRS) coordinates
+   AZ: average azimuth angle during the observarion
+   ALT: average altitude angle during the observarion
+   MUON_EFFICIENCY: average muon efficiency of the telescopes
+   TIME_START: start time of the observation stored as number of seconds after the time reference in the header
+   TIME_STOP: end time of the observation in the same forma as TIME_START
+   TIME_OBSERVATION: duration of the observation
+   TIME_LIVE: duration of the observation without dead time
+   TRIGGER_RATE: average trigger rate of the system
+   MEAN_TEMPERATURE: average temperature of the atmosphere
+   N_TELS: number of telescopes participating in the observation
+   TEL_LIST: string with a CSV list of telescope IDs participating in the observation
+   QUALITY: data quality; recommended: "spectral" or "detection" (not used by Gammapy at the moment)
+
+Extra user defined columns are allowed; Gammapy will ignore them.
+
+In order for the extra columns to have full meaning a header is needed with at least the following keywords::
+
+   OBSERVATORY_NAME: name of the observatory where the observations were taken. This is important for instance for coordinate transformations between celestial (i.e. RA/dec) and terrestrial (i.e. az/alt) coordinate systems.
+   TIME_REF_MJD_INT: reference time for other times in the list (i.e. TIME_START/TIME_STOP). Integer value in mean julian days.
+   TIME_REF_MJD_FRA: fraction of integer value defined in TIME_REF_MJD_INT.
+
+TODO: change names already defined in event lists (and in `~gammapy.utils.time.time_ref_from_dict`) ``MJDREFI`` and ``MJDREFF``? Or adopt ``TIME_REF_MJD_INT`` and ``TIME_REF_MJD_FRA``?
+
+TODO: should the observation list have already a header? Or should the header values be defined as columns, then interpreted correspondingly when stored into an ObservationTable? (I might be mixing up 2 things: obs lists and obs tables here? or should they have the same format?)
+
+TODO: should the observation list already define the times in this ``TIME_REF_MJD_INT + TIME_REF_MJD_FRA`` format? Or should it be converted once its read into an ObservationTable?
+
+TODO: ``TEL_LIST``: incompatibility wit CSV format, since it's a CSV list of tels???!!!
```

Comment:
You can have comma-separated values in a column in a CSV file. Quoting takes care of that.
But this is not very user-friedly to edit in a CSV file.
The HESS integer telescope pattern is also not very user-friendly and I don't know if it even scales to the 100 CTA telescopes. A bitpattern `100010110101` would be another option, less human-readable than the comma-separated telescope numbers, but better to store in a CSV file.

I'd say just choose something here. And please file a new issue that in the future we should add a class to convert between various formats to store TELARRAY specifications and then re-think how to store this in observation lists. Note that the HAP-exported event list files have a TELARRAY extension (see https://gammapy.readthedocs.org/en/latest/api/gammapy.data.TelescopeArray.html), you could also have a look at that and implement that functionality on that class. But maybe best to defer to a future issue / pull request?